docs/vt-extensions/vertical-line-marks.md

```diff
@@ -0,0 +1,48 @@
+# Vertical Line Markers
```

Comment:
Overlaps with `- Line marks: demo/line-marks.md` we can merge them together and reference this document in the `features.md`